app/js/view/list_view.js

```diff
@@ -82,8 +83,10 @@ export default class ListView extends View {
     var button = element.querySelector('.install-button');
     if (data.installed) {
       button.textContent = 'Open';
+      button.classList.add('open');
```

Comment:
'installed' is a better name as it describes the state. 'open' is very vague and doesn't describe state well.